README.rst

```diff
@@ -23,10 +23,10 @@
 .. |DOI| image:: https://zenodo.org/badge/21369/scikit-learn/scikit-learn.svg
 .. _DOI: https://zenodo.org/badge/latestdoi/21369/scikit-learn/scikit-learn
 
-scikit-learn
-============
+.. image:: doc/logos/scikit-learn-logo.png
```

Comment:
Would linking to http://scikit-learn.org/ be more useful than linking to the image on github?

Reply:
Added Target URL to the image